optuna_integration/botorch/botorch.py

```diff
@@ -68,6 +74,11 @@ def _get_sobol_qmc_normal_sampler(num_samples: int) -> SobolQMCNormalSampler:
     from botorch.acquisition.analytic import LogConstrainedExpectedImprovement
     from botorch.acquisition.analytic import LogExpectedImprovement
 
+with try_import() as _imports_qhvkg:
+    from botorch.acquisition.multi_objective.hypervolume_knowledge_gradient import (
```

Comment:
This was only added in a relatively recent version of BoTorch in pytorch/botorch#1950.